packages/constellation/package.json

```diff
@@ -44,6 +44,7 @@
     "@rollup/plugin-commonjs": "21.0.1",
     "@rollup/plugin-node-resolve": "13.1.2",
     "@rollup/plugin-typescript": "8.3.0",
+    "@storybook/addon-a11y": "^6.5.9",
```

Comment:
Should the version be pinned?
The other packages are

Reply:
Good call, will pin this version.